x/gov/spec/02_state.md

```diff
@@ -93,7 +93,7 @@ type ProposalStatus byte
 
 
 const (
-	StatusNil           ProposalStatus = 0x00
+	  StatusNil           ProposalStatus = 0x00
```

Comment:
```suggestion
    StatusNil           ProposalStatus = 0x00
```

Reply:
> 

sorry @blushi , I've deleted a comment of yours by mistake, saying `there is some formatting issue` here.
But I've made necessary changes.